src/clusterfuzz/_internal/bot/tasks/cron/backup.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Run the backup cron task."""
+
+import datetime
+
+from google.cloud import ndb
+from googleapiclient import discovery
+from googleapiclient import errors
+
+from clusterfuzz._internal.base import utils
+from clusterfuzz._internal.config import local_config
+from clusterfuzz._internal.metrics import logs
+
+# CrashStatistic is excluded because the number of records is too high and
+# can be rebuilt from BigQuery dataset.
+EXCLUDED_MODELS = {'CrashStatistic', 'CrashStatisticJobHistory'}
+
+
+def _datastore_client():
```

Comment:
Why are we not using the datastore helpers such as data_types and data_handler?

Reply:
This is just moving code over from the previous cron. I believe we used the datastore client directly since ndb doesn't come with the backup methods.

Reply:
ACK